### DISCUSSION TITLE
screensaver not working

### DISCUSSION THREAD

**Discussion by trieeuuu** [rocksdanister/lively#2398] [Q&A]
I've been trying to get the screensaver alone to work but it doesn't show up. It says no active wallpaper. It only works when both the main wallpaper and the screensaver has the same video on. I only want to have a live wallpaper on screensaver. Can anyone help? Thanks.

**Comment by rocksdanister** [comment]
> It says no active wallpaper

I dont remember writing this message, can you show a screenshot of this error.
